rust/foxglove/src/websocket/tests.rs

```diff
@@ -138,7 +138,7 @@ async fn test_client_connect() {
         ServerInfo::new("mock_server").with_session_id("mock_sess_id")
     );
 
-    server.stop().await;
+    server.stop();
```

Comment:
It will be really nice when we can just drop the server handle to stop the server and clean it up, we can remove all these stop calls at the end of the tests